Makefile

```diff
@@ -1 +1,3 @@
+# Generated by mirage.v4.5.1
+
 -include Makefile.user
```

Comment:
This was explicitly avoided in https://github.com/mirage/mirage-www/commit/d25d82b9aa4e36b7308985a0867dda5f1206a4e9

Having `mirage configure` override this file is a bit annoying indeed, I wonder if there is a better solution to have both workflows supported

Reply:
You’re right.
I’ve instead just removed the `Makefile` in the GHA workflow.
I’ve also added a note to that effect in the README until a better solution is found.